contracts/implementation/PoolCommitter.sol

```diff
@@ -20,9 +20,14 @@ contract PoolCommitter is IPoolCommitter, Initializable {
 
     IAutoClaim public autoClaim;
     uint128 public override updateIntervalId = 1;
+    /// ABDKMathQuad-formatted representation of the number one
+    bytes16 public constant one = 0x3fff0000000000000000000000000000;
     // The amount that is extracted from each mint and burn, being left in the pool. Given as the decimal * 10 ^ 18. For example, 60% fee is 0.6 * 10 ^ 18
-    bytes16 mintingFee;
-    bytes16 burningFee;
+    bytes16 public mintingFee;
+    bytes16 public burningFee;
+    bytes16 public changeInterval;
```

Comment:
I think a comment here would be good just to describe what exactly `changeInterval` means.